src/main/resources/templates/sites/ranking.html

```diff
@@ -96,8 +100,12 @@ <h2 th:text="#{'sites.ranking'}"></h2>
 </div>
 <!-- Pager Buttons Horizontal -->
 <div class="btn-group-horizontal hide-on-large-screen">
-  <button class="pager" onclick="previousPage()"><i class="fas fa-chevron-circle-up"></i></button>
-  <button class="pager" onclick="nextPage()"><i class="fas fa-chevron-circle-down"></i></button>
+  <div th:if="${(param.site != null) and (param.site != 0)}">
+    <button class="pager" onclick="previousPage()"><i class="fas fa-chevron-circle-up"></i></button>
+  </div>
+  <div th:if="${@rankingService.getPositionOfPlayer(users.get(users.size()-1).ranking) != @userService.getAllUsers().size()}">
```

Comment:
Can you make a call to `getUserPageSortedByRanking()` with the next page and check if it is empty? 
This check is a bit hard to understand.